firmware/application/apps/ui_scanner.hpp

```diff
@@ -130,7 +130,7 @@ class ScannerView : public View {
 
 	void on_statistics_update(const ChannelStatistics& statistics);
 	void on_headphone_volume_changed(int32_t v);
-	void handle_retune(uint32_t i);
+	void handle_retune( long i );
```

Comment:
Is this change necesssary ? the value stored in handle_retune is uint32_t.

Reply:
I had a problem with it being an uint32_t => only up to 4294967295 (4.29 GHZ)
I tried with a long but it was not enough as a long is also an int32_t. It's all fixed in my repo, I thought I made the commit, but it looks like I didn't.
The fix was to use an int64_t.

Reply:
Do you want me to add latest search files to the PR ?

Reply:
Maybe, use rf::Frequency so we adhere at the convention used in most other apps

Reply:
That's what I'm doing actually, but it's not yet ready nor uploaded to git.
I'm planning on just using the array returned by load_freqman_file as it is.
As I'm also working on freqman it's all somewhat related.

Reply:
In my scanner code, I use the message.range to pass the frequency_index (which is 0 to 500, so more than OK to use uint32_t)

Reply:
And message is returning a Frequency, which is defined in rf_path.hpp as Frequency = int64_t;